src/main/scala/com/cognite/sdk/scala/v1/resources/sessions.scala

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1.resources
+
+import com.cognite.sdk.scala.common._
+import com.cognite.sdk.scala.v1._
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder, Printer}
+import sttp.client3._
+import sttp.client3.circe._
+
+class Sessions[F[_]](val requestSession: RequestSession[F])
+    extends WithRequestSession[F]
+    with BaseUrl {
+  import Sessions._
+  override val baseUrl = uri"${requestSession.baseUrl}/sessions"
+
+  def createWithTokenExchangeFlow(items: Items[SessionCreateWithToken]): F[Seq[Session]] = {
+    implicit val customPrinter: Printer = Printer.noSpaces.copy(dropNullValues = true)
+    requestSession.post[Seq[Session], Items[Session], Items[SessionCreateWithToken]](
+      items,
+      baseUrl,
+      value => value.items
+    )
+  }
+
+  def createWithClientCredentialFlow(items: Items[SessionCreateWithCredential]): F[Seq[Session]] = {
+    implicit val customPrinter: Printer = Printer.noSpaces.copy(dropNullValues = true)
+    requestSession.post[Seq[Session], Items[Session], Items[SessionCreateWithCredential]](
+      items,
+      baseUrl,
+      value => value.items
+    )
+  }
+
+  def listAll(): F[Seq[SessionList]] =
```

Comment:
Other resources use `list()`, are we replicating the sessions api naming here, or can we rename this one to list as well?

Reply:
![image](https://user-images.githubusercontent.com/92026183/139871605-5e612dcd-304f-4d92-b7a0-c8d43e8609e9.png)